### ISSUE TITLE
Can't control Scroll Zone lightning on Corsair M65 RGB ELITE

### ISSUE THREAD

**Issue by Xeevis** [DarthAffe/RGB.NET#97]
When I try to set all Corsair LEDs to red, it works for whole **K95 RGB PLATINUM**, but only for logo zone of **M65 RGB ELITE**, for scroll zone lightning is unaffected.

![image](https://user-images.githubusercontent.com/5835044/71110715-1fdd6400-21c8-11ea-8e88-a94b44a195fb.png)

```csharp
surface.LoadDevices(CorsairDeviceProvider.Instance);

var allLeds = new ListLedGroup(surface.Leds)
{
    Brush = new SolidColorBrush(new Color(255, 0, 0))
};

surface.Update();
```

**Comment by Xeevis** [comment]
Remapping `LedId.Mouse2` to `CorsairLedId.B3` solves the issue.

**Comment by DarthAffe** [comment]
Fixed in #99
